### DISCUSSION TITLE
Permissions for Hosted Developer Portal

### DISCUSSION THREAD

**Discussion by cevich** [renovatebot/renovate#30964] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I'm assuming a user's GitHub org. and/or repo permissions are consulted WRT granting access to view renovate logs on [the developer portal](https://developer.mend.io/).  However, searching around docs and discussions, I couldn't find anywhere mentioning what permissions are consulted/required.  Specifically, I have a new team member that can only see _some_ of the org's repos in the portal.  Other than giving him org-admin, what permissions does he need on a repo-by-repo level to be able to view renovate logs?

**Comment by rarkins** [comment, marked as answer]
We give repo log access to those who have write access. If the developer has only read repo access then that would explain your situation.

**Reply by cevich** [reply]
Excellent, that's exactly what I needed to know.  Thanks!
